lib/bang.coffee

```diff
@@ -101,13 +104,14 @@ module.exports = Bang =
       # it's a new buffer
       cwd = @referenceDir()
     # The text to give as input to the command
-    input = editor?.getSelectedText()
+    input = editor?.getSelectedText()||editor?.getText()
```

Comment:
Sometimes a command may behave different if there is input or not. It might be confusing for the user not selecting any text but giving the whole buffer as input to the command.